src/voting-contracts/contracts/IStrategies.sol

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+interface IStrategies {
+    function evaluateVotingPower(address _voter) external returns (uint256);
```

Comment:
In solidity as per Openzepplin [see here](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC20/ERC20.sol)normally only private variables or functions are prefixed with underscore **_** and not function